### PR TITLE
Revamp formatter script

### PR DESCRIPTION
**Pull request by adambujak**
- Separate into docker and non docker script
    - Add blacklist file

### CODE REVIEW

**Comment by harneetMac** [comment]
I am getting this error message:
image

Am I missing something?

**Comment by adambujak** [comment]
> I am getting this error message: image
> 
> Am I missing something?

what did you run to get this?

`make format`?

**Comment by stephen99scott** [comment]
> > I am getting this error message: image
> > Am I missing something?
> 
> what did you run to get this?
> 
> `make format`?

Yeah make format gives a similar error for me in MingW 
image

**Comment by adambujak** [comment]
can you guys retry this now

File: .gitattributes [hunk 1]
```diff
@@ -1 +0,0 @@
-* text=auto
```
Comment (stephen99scott) [inline .gitattributes:1]: I don't think this is necessary

**Comment by stephen99scott** [comment]
It's still giving the same error for me

**Comment by adambujak** [comment]
> It's still giving the same error for me

if you run `dos2unix` on `tools/formatter_blacklist.txt` and `tools/run_formatter.sh` it fixes the error

idk why this is the case

**Review by stephen99scott — APPROVED** [review]
Works as expected now. Small detail should we hide this line:
image

**Pull request merged**